test/hcp/testdata/vm.golden

```diff
@@ -114,7 +116,8 @@ resource "hcp_consul_cluster_root_token" "token" {
 }
 
 module "vm_client" {
-  source = "../../modules/hcp-vm-client"
+  source  = "hashicorp/hcp-consul/azurerm//modules/hcp-vm-client"
```

Comment:
Double slash?

Reply:
That is actually required: https://www.terraform.io/language/modules/sources#modules-in-package-sub-directories